docs/root/intro/arch_overview/upstream/load_balancing/load_balancers.rst

```diff
@@ -132,16 +132,18 @@ host B has a weight of 2, then host A will have 21,846 entries and host B will h
 regardless of the configured host and locality weights, so in some extreme cases the actual
 proportions may differ from the configured weights. For example, if the total number of hosts is
 larger than the fixed table size, then some hosts will get 1 entry each and the rest will get 0,
-regardless of weight. Best practice is to monitor the :ref:`min_entries_per_host and
-max_entries_per_host gauges <config_cluster_manager_cluster_stats_maglev_lb>` to ensure no hosts
-are underrepresented or missing.
+regardless of weight.
 
 In general, when compared to the ring hash ("ketama") algorithm, Maglev has substantially faster
 table lookup build times as well as host selection times (approximately 10x and 5x respectively
 when using a large ring size of 256K entries). The downside of Maglev is that it is not as stable
 as ring hash. More keys will move position when hosts are removed (simulations show approximately
-double the keys will move). With that said, for many applications including Redis, Maglev is very
-likely a superior drop in replacement for ring hash. The advanced reader can use
+double the keys will move). Maglev aims for minimal disruption when set of upstream hosts change.
+Best practice is to set
+:ref:`table_size<envoy_v3_api_field_config.cluster.v3.Cluster.MaglevLbConfig.table_size>` and monitor
+the :ref:`min_entries_per_host and max_entries_per_host gauges <config_cluster_manager_cluster_stats_maglev_lb>`
+to ensure no hosts are underrepresented or missing. With that said, for many applications
```

Comment:
```suggestion
to ensure no hosts are under-represented or missing. With that said, for many applications
```

seems this can be spelt both ways - but i think its more readable this way